lib/ui/isolate_name_server/isolate_name_server.h

```diff
@@ -9,11 +9,21 @@
 #include <mutex>
 #include <string>
 
+#include "flutter/fml/build_config.h"
 #include "flutter/fml/synchronization/thread_annotations.h"
 #include "lib/fxl/macros.h"
 #include "third_party/dart/runtime/include/dart_api.h"
 
+// The Android NDK's headers don't have the clang annotations.
+#if !defined(OS_ANDROID)
```

Comment:
Should this be centralized in fml/synchronization/thread_annotations.h?

@chinmaygarde